customer-terminate-service/app/src/main/java/se/inera/intyg/cts/application/api/TerminationController.java

```diff
@@ -59,4 +60,22 @@ TerminationDTO startErase(@PathVariable UUID terminationId) {
       throw new ResponseStatusException(HttpStatus.BAD_REQUEST, ex.getMessage(), ex);
     }
   }
+
+  /**
```

Comment:
Javadoc that doesn't add any additional value/description than the method name itself is redundant and create noise. I think your code is self-documenting.

Reply:
I prefere javadoc as best practise and have never felt that it create noise. All external libraries does contain it and i find it helpfull that you can get a small description just by howering over the classname or method. By experience self-documented methods tends to not be self-documented in the long run and/or be self documented to the one who writes them. :)